src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -102,16 +105,31 @@ const AttendanceFormBuilder = () => {
     };
 
     fetchAttendance();
-  }, [paramId, form]);
+  }, [attendanceId, form]);
 
   const onSubmit = async (values: AttendanceFormValues) => {
+    const updatedValue = {
+      id:attendanceId,
+      ...values
+     }
     try {
-      const result = await createAttendance(values);
-      if (!result) {
-        toast({
-          title: "Attendance submit successfully",
-          description: "You will receive message shortly",
-        });
+      if (attendanceId) {
+        const updatedResult = await updateAttendance(updatedValue);
+
+        if (updatedResult) {
```

Comment:
i think from action only error is returning so we need `!` operator here for toaster because in error case it will go to catch section?

Reply:
Fixed